pkg/retry/retry.go

```diff
@@ -40,7 +40,7 @@ func (e *ErrorSet) Error() string {
 
 	errString := fmt.Sprintf("%d error(s) occurred:", len(e.errs))
 	for _, err := range e.errs {
-		errString = fmt.Sprintf("%s\n%s", errString, err)
+		errString = fmt.Sprintf("%s\n\t%s", errString, err)
```

Comment:
This may need an update to the tests.

Reply:
oh yeah, totally forgot, thanks, I'll get it updated